EXP-Client/todo/Todo.js

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import { Provider } from 'react-native-paper';
+import {
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native';
+import ProgressCircle from 'react-native-progress/Circle';
+import AddTaskMenu from './AddTaskMenu';
+import TaskList from './TaskList';
+
+function Graph({ tasks }) {
+  const completedCount = 2; // tasks.reduce(({ name, deadline, completed }, count) => completed ? count + 1 : count, 0);
+  return (
+    <View style={styles.graph}>
+      <ProgressCircle
+        size={80}
+        progress={completedCount / tasks.length}
+        showsText={true}
+        formatText={() => `${Math.round((completedCount / tasks.length) * 100)}%`}
+        style={styles.progressCircle}
+      >
+        <Text style={styles.progressText}>April 1, 2023</Text>
+      </ProgressCircle>
+    </View>
+  );
+}
+
+function HeaderText({ tasks }) {
+  return (
+    <View style={styles.headerText}>
+      <Text style={{ fontSize: 20 }}>Today you have...</Text>
+      {tasks.map(({ name, deadline, completed }) =>
```

Comment:
From a design standpoint, I found listing the tasks in the header as well kind of redundant since the list is right underneath.

Reply:
I'm fine with this change if everyone else in the group agrees